library/src/compute/calc.rs

```diff
@@ -530,7 +530,7 @@ fn binom(n: u64, k: u64) -> u64 {
     let mut result = 1;
 
     for i in 0..real_k {
-        result = result * (n - i) / (i + 1)
+        result *= (n - i) / (i + 1)
```

Comment:
These statements are not equivalent.
`binom(6, 2)` returns 12 instead of 15.

Reply:
It should be good now, thank you!